### ISSUE TITLE
Overloads with argument lists

### ISSUE THREAD

**Issue by adamralph** [adamralph/simple-exec#430]
e.g. in addition to:

```c#
Run("foo.exe", "arg1 arg2");
```

you can now do:

```c#
Run("foo.exe", new[] { "arg1", "arg2" });
```

**Comment by adamralph** [comment]
changes in #425 released in [9.1.0-alpha.1](https://www.nuget.org/packages/SimpleExec/9.1.0-alpha.1)

**Comment by adamralph** [comment]
Currently these overloads do not echo. Since the argument escaping is done by `System.Diagnostics`, forming the command line is not just a case of concatenating the args, since they are not escaped. Perhaps the command should be echoed in a "deconstructed" way, e.g.

```text
SimpleExec: Name: foo
SimpleExec: Arg 1: bar
SimpleExec: Arg 2: baz
```

**Comment by adamralph** [comment]
re-released in [9.1.0-rc.1](https://www.nuget.org/packages/SimpleExec/9.1.0-rc.1)
